functions/jens.js

```diff
@@ -49,7 +49,8 @@ const makeSendMessage = (sendData) => {
 Route: ${sendData.zlaggableName}
 Crag: ${sendData.cragName}
 Area: ${sendData.areaName}
-Difficulty: ${sendData.difficulty}`
+Difficulty: ${sendData.difficulty}
+Comment: ${sendData.comment}`
```

Comment:
Should we chunk the string so we get n 100 char length strings?
```suggestion
Comment: ${sendData.comment}.match(/(.|[\r\n]){1,100}/g).join("\n")`
```

Reply:
neato regex! looks like a good solution

Reply:
though, it looks like there's an error in your template literal: sb - ${sendData.comment.match(/(.|[\r\n]){1,100}/g).join("\n")}` @MasonCaiby

Reply:
ah yeah. good catch. incomment coding is always dangerous. The main downside here is that we'll split a word if it hits char 100. I guess we could split the str on ' ' then just get join every 20th word with "\n" and the others with "\ ". Not sure if JS has a more optimal str chunker though?

Reply:
How important do you think this is? Maybe the slack message will be relatively well formatted anyways, especially if the comment is always at the end

Reply:
Yeah, idk if comment cleaning is needed. Maybe we just add the comment as it comes to us and revisit later? I guess most users post shorter comments anyway...